test.sh

```
@@ -0,0 +1,28 @@
+
+ hf download   inference-optimization/gemma4-31B-responses magpie_gemma-4-31B-it.jsonl   --repo-type dataset   --local-dir ./output/dataset
+
+ python scripts/prepare_data.py   --model google/gemma-4-31B-it   --data ./output/dataset/magpie_gemma-4-31B-it.jsonl   --output ./output/mtp_gemma-4-31B-it   --max-samples 5000   --seq-length 8192
+
+CUDA_VISIBLE_DEVICES=0,1 python scripts/launch_vllm.py   google/gemma-4-31B-it --hidden-states-path /tmp/hidden_states_$(whoami )-- --port 8000  --tensor-parallel-size 2 --max-model-len 8192
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)